plugins/funind/g_indfun.mlg

```diff
@@ -264,7 +264,7 @@ VERNAC COMMAND EXTEND NewFunctionalCase
 END
 
 (***** debug only ***)
-VERNAC COMMAND EXTEND GenerateGraph CLASSIFIED AS QUERY
-| ![ proof ] ["Generate" "graph" "for" reference(c)] ->
-  { make_graph (Smartlocate.global_with_alias c) }
+VERNAC COMMAND EXTEND GenerateGraph CLASSIFIED AS QUERY STATE MAYBEOPENPROOF
```

Comment:
>    CLASSIFY AS QUERY together with ![proof w].

So this?

Reply:
I'm not sure if it really opens a proof, that's just what the type ended up as.

Reply:
It is for sure not a QUERY, since it generates terms. I don't think it can leave a proof open.